src/schema.patch

```diff
@@ -0,0 +1,131 @@
+diff --git a/src/schema.rs b/src/schema.rs
+index d0d6a3b..9d4f5d1 100644
+--- a/src/schema.rs
++++ b/src/schema.rs
+@@ -30,13 +30,12 @@ table! {
```

Comment:
Let's add the comment here, something like "this file is autogenerated, consider edit diesel.toml"?

Sorry again that I edited `schema.rs` manually 🙂